File: B74560/Laboratorios/Lab3/Orbis/src/main/java/com/orbis/B74560/business/LibroBusiness.java
package com.orbis.B74560.business;

import java.util.Iterator;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import com.orbis.B74560.data.LibroData;
import com.orbis.B74560.domain.Libro;
import com.orbis.B74560.domain.LibroAutor;

@Service
public class LibroBusiness {

	@Autowired
	private LibroData libroDao;

	public Iterator<Libro> buscarPorEditorial(int idEditorial) {
		return libroDao.buscarPorEditorial(idEditorial);
	}

	public void guardarLibroAutor(Libro libro, LibroAutor libroAutor) {
		libroDao.guardarLibro(libro, libroAutor);
	}
	
}

File: B62734/lab4/Orbis[B62734].Model/Business/LibroBusiness.cs
using Orbis_B62734_.Model.Data;
using Orbis_B62734_.Model.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace Orbis_B62734_.Model.Business
{
   public class LibroBusiness
    {
        LibroData libroData;

        public LibroBusiness()
        {
            libroData = new LibroData();
        }
        public IEnumerable<Libro> GetByCodPublicador(int codPublicador) {
            yield return libroData.GetByCodPublicador(codPublicador);
        }
    }
}
